pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools >=42", "setuptools_scm[toml] >=3.4.1", "wheel"]
+requires = ["setuptools >=42", "setuptools_scm[toml] >=3.4.1"]
```

Comment:
i suspect a version pin increase for setuptools is also necessary, can we crosscheck